charts/skypilot/templates/dcgm-cluster-filtering-dashboard-grafana-configmap.yaml

```
@@ -0,0 +1,13 @@
+{{- if and .Values.grafana.enabled .Values.grafana.sidecar.dashboards.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-dcgm-grafana-dashboard
```

Should we rename this to something more representative:
```suggestion
  name: {{ .Release.Name }}-dcgm-skycluster-dashboard
```
May have to rename the file and the json too